wgs/wgs-gather-vcfs/README.md

```diff
@@ -0,0 +1,36 @@
+# WGSGatherVcfs with GATK
+
+This workflow runs the GATK's `GatherVcfsCloud` on a GCS directory containing sharded VCF files from a joint genotyping workflow such as the one here: https://github.com/gatk-workflows/gatk4-germline-snps-indels.
+
+Each VCF file corresponds to an interval from the joint genotyping intervals file. That file is a required input to this workflow.
+
+The output of the workflow is 1 VCF file per chromosome requested.
+
+Inputs:
+- intervals_file: GCS path of intervals file
+- chromosomes: An array of chromosome string values, ["1", "2", "21", "22", "X", "Y"]
+- shards_to_ignore: An array of numbers from the intervals file to ignore.
+                    A few intervals from joint genotyping may have zero variants, which results in "empty" shards.
```

Comment:
I suggest removing the part:

```
, which results in "empty" shards.
```

as that would mislead people to look for 0 length files.

Let's just say:

```
Some intervals from joint genotyping may have zero variants called.
List these shards here to prevent GatherVcfs from raising an error.
```